main/resource-access.bs

```diff
@@ -95,6 +95,44 @@ Behaviour pertaining to authorization MUST proceed this optional redirect
 
 ## Linked Data ## {#linked-data}
 
+### Storage ### {#storage}
```

Comment:
Note: I was initially confused by the term, thinking that it referred to backends. Perhaps a small clarification about what it is and is not. The definition "a storage is the root container for all of its contained" is technically correct, but does not inform people who have never seen `pim:Storage`.

Reply:
Updated with https://github.com/solid/specification/pull/208/commits/cd78a59126171ca27e593b51180353ae8044145f . May be a bit redundant but readable. Thanks.